invenio_rdm_records/oai.py

```diff
@@ -27,6 +30,46 @@ def dublincore_etree(pid, record):
     return DublinCoreXMLSerializer().serialize_object_xml(record["_source"])
 
 
+def datacite_etree(pid, record):
+    """Dump the record (search result) into a DataCite XML format for OAI-PMH."""
+    data_dict = DataCite43XMLSerializer().dump_one(record["_source"])
+    return schema43.dump_etree(data_dict)
+
+
+def oai_datacite_etree(pid, record):
+    """Dump the record (search result) into a OAI DataCite XML format for OAI-PMH."""
+    datacentre_symbol_text = None
+    resource_dict = DataCite43XMLSerializer().dump_one(record["_source"])
+
+    nsmap = {
+        None: "http://schema.datacite.org/oai/oai-1.1/",
+        "xsi": "http://www.w3.org/2001/XMLSchema-instance",
+    }
+
+    attribs = {
+        f"{{{nsmap['xsi']}}}schemaLocation": "http://schema.datacite.org/"
+                                             "oai/oai-1.1/ "
+                                             "http://schema.datacite.org/"
+                                             "oai/oai-1.1/oai.xsd",
```

Comment:
```suggestion
        f"{{{nsmap['xsi']}}}schemaLocation": (
            "http://schema.datacite.org/oai/oai-1.1/ "
            "http://schema.datacite.org/oai/oai-1.1/oai.xsd"
        ),
```

I tried with `black` as well, but didn't get anything more intuitive. So in the end "hand-splitted" looks the most logical I think.